tests/virt_autotest/guest_upgrade_run.pm

```diff
@@ -18,19 +18,25 @@ use warnings;
 use base "virt_autotest_base";
 use virt_utils;
 use testapi;
+use Utils::Architectures;
 
 sub get_script_run {
     my $pre_test_cmd         = "/usr/share/qa/tools/test_virtualization-guest-upgrade-run";
     my $product_upgrade_repo = get_var("PRODUCT_UPGRADE_REPO", "");
     my $max_test_time        = get_var("MAX_TEST_TIME", "36000");
+    my $vm_xml_dir           = "/tmp/download_vm_xml";
 
     handle_sp_in_settings_with_sp0("PRODUCT_UPGRADE");
-    my $product_upgrade = get_var("PRODUCT_UPGRADE", "sles-12-sp2-64");
+    my $product_upgrade = get_required_var("PRODUCT_UPGRADE");
 
     handle_sp_in_settings_with_fcs("GUEST_LIST");
-    my $guest_list = get_var("GUEST_LIST", "sles-12-sp1-64");
+    my $guest_list = get_required_var("GUEST_LIST");
+    $guest_list =~ s/(sles-\d+)-sp0/$1-fcs/i;
 
```

Comment:
Why not s/sp0/fcs/ directly?

Reply:
I thought it is more strict. but it makes code a little complex to be read. To make it simple is ok.